src/scripts/main.js

```diff
@@ -1,7 +1,229 @@
 'use strict';
+/* eslint-disable */
+const refs = {
+  startBtn: document.querySelector('.start'),
+  fieldCells: document.querySelectorAll('.field-cell'),
+  fieldRows: document.querySelectorAll('.field-row'),
+  messageStart: document.querySelector('.message-start'),
+  messageLose: document.querySelector('.message-lose'),
+  messageWin: document.querySelector('.message-win'),
+  score: document.querySelector('.game-score'),
+};
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
```

Comment:
You had to do it via Game class
Now all tests fail

But it works good, so approved 👍